projects/aws/eks-a-admin-image/provisioners/download_eksa_artifacts.sh

```diff
@@ -9,10 +9,12 @@ EKSA_MANIFESTS_PATH="${EKSA_MANIFESTS_PATH:-/usr/lib/eks-a/manifests}" # The ful
 EKSA_ARTIFACTS_TAR_PATH="${EKSA_ARTIFACTS_TAR_PATH:-/usr/lib/eks-a/artifacts/artifacts.tar}" # The path where the tar containing all eks-a container artifacts will be stored
 
 sudo mkdir -p $EKSA_MANIFESTS_PATH
-sudo eksctl anywhere download artifacts --retain-dir --download-dir $EKSA_MANIFESTS_PATH -v4
+sudo -E env "PATH=$PATH" eksctl anywhere download artifacts --retain-dir --download-dir $EKSA_MANIFESTS_PATH -v4
 sudo chmod -R a+r $EKSA_MANIFESTS_PATH
 
 ARTIFACTS_NAME_DIR=$(dirname $EKSA_ARTIFACTS_TAR_PATH)
 sudo mkdir -p $ARTIFACTS_NAME_DIR
-sudo eksctl anywhere download images -o $EKSA_ARTIFACTS_TAR_PATH -v4
+sudo -E env "PATH=$PATH" eksctl anywhere download images -o $EKSA_ARTIFACTS_TAR_PATH -v4
 sudo chmod -R a+r $ARTIFACTS_NAME_DIR
+
+sudo docker system prune --volumes --force
```

Comment:
super nit: add a comment to explain why we do this pls pls

Reply:
Done!